example/ios/reactnativeyocoexample/Info.plist

```diff
@@ -74,5 +74,21 @@
     <string>Light</string>
     <key>UIViewControllerBasedStatusBarAppearance</key>
     <false/>
+    <key>NSBluetoothAlwaysUsageDescription</key>
```

Comment:
I think we should use `app.json` expo ios permissions and do `npx expo prebuild` to automatically add these lines?